OSDOCS-13985: updates multus in config.yaml MicroShift (#91783)
openshift/openshift-docs: modules/microshift-config-parameters-table.adoc

|IP address block
|A block of IP addresses from which pod IP addresses are allocated. IPv4 is the default network. Dual-stack entries are supported. The first entry in this field is immutable after {microshift-short} starts. Default range is `10.42.0.0/16`.

|`network.multus.status`
|`string`

> |Controls the deployment of the Multus Container Network Interface (CNI). Default status is `Disabled`. After being set to `Enabled`, the Multus CNI cannot be deleted.

Reviewer: To avoid the passive voice in the first half of the sentence:
```suggestion
|Controls the deployment of the Multus Container Network Interface (CNI). Default status is `Disabled`. If you set the value to `Enabled`, the Multus CNI cannot be deleted.
```
